lib/importers/daily_calls/tp/retriever.rb

```diff
@@ -0,0 +1,31 @@
+require 'mail_retriever'
+
+module Importers
+  module DailyCalls
+    module TP
+      class Retriever
+        class MissingProcessingBlock < StandardError; end
```

Comment:
Can we suffix this class name with `Error`, I don't tend to include pattern language in class names but it's idiomatic Ruby to suffix error / exception classes respectively.